substratest/settings.py

```diff
@@ -13,13 +13,11 @@
 _DEFAULT_NETWORK_CONFIGURATION_PATH = os.path.join(_CURRENT_DIR, "../", "values.yaml")
 _SUBSTRA_TESTS_CONFIG_FILEPATH = os.getenv("SUBSTRA_TESTS_CONFIG_FILEPATH", _DEFAULT_NETWORK_CONFIGURATION_PATH)
 
-_DEFAULT_SUBSTRA_TOOLS_TAG_LOCAL = (
-    f"latest-nvidiacuda11.8.0-base-ubuntu22.04-python{sys.version_info.major}.{sys.version_info.minor}"
-)
-_DEFAULT_SUBSTRA_TOOLS_TAG_REMOTE = "latest"
+_DEFAULT_DOCKER_TAG_LOCAL = f"{sys.version_info.major}.{sys.version_info.minor}-slim"
+_DEFAULT_DOCKER_TAG_REMOTE = "3.11-slim"
```

Comment:
Why do we need these two version, won't `_DEFAULT_DOCKER_TAG_LOCAL` return `3.11-slim` if started from the correct Python version? 

I can see that before, there was a difference in path between the two, but it looks like to me that we could use this opportunity to simplify the code